client/src/App.js

```diff
@@ -23,6 +23,7 @@ class App extends Component {
           <h2>Welcome to React</h2>
           {this.state.text}
         </div>
+        <MusicApps />
```

Comment:
Move MusicApps below Top Charts. I think TopCharts is one of the first things in the mock

Reply:
top charts is actually one of the last things in the mock, music apps are above it